workers/unity/Packages/com.improbable.gdk.deploymentlauncher/Commands/WrappedTask.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Improbable.Gdk.DeploymentManager.Commands
 {
-    public class WrappedTask<TResult, TContext> : IDisposable
+    public class WrappedTask<TResult, TContext> : IDisposable, IWrappedTask
```

Comment:
Should have docstrings

Reply:
Hmmm this should probably be internal until otherwise needed actually, but yes also doc strings